README.md

```diff
@@ -1,75 +1,83 @@
-# QHub
+# `qhub` - automated data science environments on cloud environments
```

Comment:
Since, we call it QHub officially, it makes sense to have QHub here. For all the cases where we are referring to cli below, `qhub` is fine